packages/nuxt/src/pages/module.ts

```diff
@@ -38,10 +38,10 @@ export default defineNuxtModule({
       // Add default options
       context.files.push({ path: resolve(runtimeDir, 'router.options'), optional: true })
 
-      await Promise.all(nuxt.options._layers.map(async layer => {
+      for (const layer of nuxt.options._layers) {
```

Comment:
Couldn't we use await Promise.all and then loop over the resolved promises?

Reply:
Oops, yes, we could indeed.